components/inputs/input-date-time.js

```diff
@@ -146,9 +152,9 @@ class InputDateTime extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(R
 					min-value="${ifDefined(this._minValueLocalized)}"
 					?required="${this.required}"
 					?skeleton="${this.skeleton}"
+					style="${styleMap(dateStyle)}"
 					.value="${this._parsedDateTime}">
-				</d2l-input-date>
-				<d2l-input-time
+				</d2l-input-date><d2l-input-time
```

Comment:
There was some spacing being added that is no longer present if these are on the same line🤷‍♀️